common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -704,9 +710,13 @@ protected boolean isWebCpInWebviewFeatureEnabled(@NonNull final String originalU
         final String methodTag = TAG + ":isWebCpInWebviewFeatureEnabled";
         try {
             if (!ProcessUtil.isRunningOnAuthService(getActivity().getApplicationContext())) {
-                // Enabling webcp in webview feature for brokered flows only for now.
-                Logger.info(methodTag, "Not running on AuthService, skipping WebCP in WebView feature check.");
-                return false;
+                if (mIsWebViewWebCpEnabledInBrokerlessCase) {
```

Comment:
Why don't we assign mIsWebViewWebCpEnabledInBrokerlessCase to mInWebCpFlow and then log the status?